src/ucp/proto/proto_multi.c

```diff
@@ -195,6 +197,17 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
         mpriv->max_frag_sum += lpriv->max_frag;
         lpriv->weight_sum    = weight_sum;
         lpriv->max_frag_sum  = mpriv->max_frag_sum;
+
+        iface_attr = ucp_proto_common_get_iface_attr(&params->super.super,
+                                                     lpriv->super.lane);
```

Comment:
instead of calling ucp_proto_common_get_iface_attr here, need to return opt_zcopy_align in lanes_perf as part of ucp_proto_common_get_lane_perf, since this function retrieves the low-level interface information.